euth/contrib/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from django.urls import reverse
+
+from adhocracy4.modules import models
+from adhocracy4.phases import admin as phase_admin
+from adhocracy4.projects.admin import ProjectAdminFilter
+
+
+class ProjectFilter(ProjectAdminFilter):
+    project_key = 'module__project'
+
+
+class ItemAdmin(admin.ModelAdmin):
+    list_filter = (
+        'module__project__organisation',
+        'module__project__is_archived',
+        ProjectFilter
+    )
+    list_display = ('__str__', 'creator', 'created')
+    readonly_fields = ('creator',)
+    date_hierarchy = 'created'
```

Comment:
We don't need to copy the ItemAdmin, as this is not used in the ModuleAdmin. We use it in some projects (see mB) to display and filter the items.

Reply:
I see :)